toolkit/tools/pkg/imagecustomizerlib/customizeverity_test.go

```diff
@@ -144,11 +148,38 @@ func testCustomizeImageVerityShrinkExtractHelper(t *testing.T, testName string,
 		return
 	}
 
-	// Attach partition files.
+	espPartitionPath := filepath.Join(testTempDir, fmt.Sprintf("image_%d.raw", espPartitionNum))
 	bootPartitionPath := filepath.Join(testTempDir, fmt.Sprintf("image_%d.raw", bootPartitionNum))
 	rootPartitionPath := filepath.Join(testTempDir, fmt.Sprintf("image_%d.raw", rootPartitionNum))
 	hashPartitionPath := filepath.Join(testTempDir, fmt.Sprintf("image_%d.raw", hashPartitionNum))
+	varPartitionPath := filepath.Join(testTempDir, fmt.Sprintf("image_%d.raw", varPartitionNum))
+
+	espStat, err := os.Stat(espPartitionPath)
+	assert.NoError(t, err)
+
+	bootStat, err := os.Stat(bootPartitionPath)
+	assert.NoError(t, err)
+
+	rootStat, err := os.Stat(rootPartitionPath)
+	assert.NoError(t, err)
+
+	hashStat, err := os.Stat(hashPartitionPath)
+	assert.NoError(t, err)
 
+	varStat, err := os.Stat(varPartitionPath)
+	assert.NoError(t, err)
+
+	// Check partition sizes.
+	assert.Equal(t, int64(8*diskutils.MiB), espStat.Size())
+	assert.Equal(t, int64(100*diskutils.MiB), hashStat.Size())
+
+	// These partitions are shrunk.
+	// So, their size will vary slightly.
+	assert.Greater(t, int64(100*diskutils.MiB), bootStat.Size())
```

Comment:
where do the 100, 650, 150 numbers come from?

Reply:
They are somewhat arbitrary values that make the test work. I've added a comment.